Dockerfile

```
@@ -0,0 +1,21 @@
+FROM ubuntu:22.04
+
+COPY inittimezone /usr/local/bin/inittimezone
+RUN chmod +x /usr/local/bin/inittimezone
+CMD ./usr/local/bin/inittimezone
```

```suggestion
RUN ./usr/local/bin/inittimezone
```
Otherwise, I get a "inittimezone not found"

and you want to run this while building the image (before `apt update`), not when creating the container.